packages/dicomImageLoader/src/shared/decoders/decodeHTJ2K.ts

```diff
@@ -57,8 +57,7 @@ export function initialize(decodeConfig?: LoaderDecodeOptions): Promise<void> {
 // https://github.com/chafey/openjpegjs/blob/master/test/browser/index.html
 async function decodeAsync(compressedImageFrame: ByteArray, imageInfo) {
   await initialize();
-  // const decoder = local.decoder; // This is much slower for some reason
-  const decoder = new local.codec.HTJ2KDecoder();
+  const decoder = local.decoder;
```

Comment:
This was the original implementation, but it causes a bug where some images can't be decoded and the decoding gets slower and slower over time.
The problem is that some of the state in the decoder isn't being cleared correctly.  I would try updating the openjph library to latest, make sure the malloc library used is the same one we use for openjpeg, and rebuild that part of the system.  Then you can test to see if it works using the CS3D progressive loading tests.    Test the decompression time for the second decompression, and then for the last decompression time on the HTJ2K volume, as well as running the HTJ2K stack tests, with multiple runs through the stack.  The decompression time should stay roughly constant.